packages/paste-core/components/form/src/Radio.tsx

```diff
@@ -1,59 +1,76 @@
 import * as React from 'react';
 import * as PropTypes from 'prop-types';
 import {useUID} from 'react-uid';
-import {styled, css} from '@twilio-paste/styling-library';
 import {Box} from '@twilio-paste/box';
 import {
   BaseRadioCheckboxControl,
   BaseRadioCheckboxLabel,
   BaseRadioCheckboxLabelText,
   BaseRadioCheckboxHelpText,
 } from './shared/BaseRadioCheckbox';
+import {RadioContext} from './RadioContext';
 
 export interface RadioProps extends React.InputHTMLAttributes<HTMLInputElement> {
-  children: NonNullable<React.ReactNode>;
-  hasError?: boolean;
-  helpText?: string | React.ReactNode;
   id: string;
   value: string;
+  name?: string;
+  checked?: boolean;
+  disabled?: boolean;
+  hasError?: boolean;
+  helpText?: string | React.ReactNode;
+  onChange?: (event: React.ChangeEvent<HTMLInputElement>) => void;
+  children: NonNullable<React.ReactNode>;
 }
 /* eslint-disable emotion/syntax-preference */
-const HiddenRadio = styled.input(
-  css({
-    border: '0',
-    clip: 'rect(0 0 0 0)',
-    height: '1px',
-    margin: '-1px',
-    overflow: 'hidden',
-    padding: '0',
-    position: 'absolute',
-    textTransform: 'none',
-    whiteSpace: 'nowrap',
-    width: '1px',
-  })
+type HiddenRadioProps = Pick<RadioProps, 'checked' | 'value' | 'id' | 'disabled' | 'name' | 'onChange'> & {
+  ref?: any | undefined;
+};
+const HiddenRadio: React.FC<HiddenRadioProps> = props => (
```

Comment:
Now this is a functional component do you need to forward the ref to `Box` here too?